src/tests/unit/tests/background/actions/card-selection-action-creator.test.ts

```diff
@@ -38,6 +38,42 @@ describe('CardSelectionActionCreator', () => {
         );
     });
 
+    test('onToggleVisualHelper', () => {
+        const actionMock = createActionMock(null);
```

Comment:
The name of the`const` seems very generic. I notice previously written tests use the name of the action in the mock name, e.g `toggleCardSelectionMock`. Should we do the same here to improve readability and consistency?

Reply:
I'm not for it because I think it's not really a specific action's mock at the moment but consistency is a good reason so I'll update it 😄